t/t0091-sq-quote-buf-pretty.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
```

Comment:
```
git chmod a+x t/t0091-sq-quote-buf-pretty.sh
git commit -a --amend
```

Should fix your build break.

Reply:
IGNORE ABOVE I'M AN IDIOT:

```
git add --chmod=a+x t/t0091-sq-quote-buf-pretty.sh
git commit --amend
```

https://git-scm.com/docs/git-add#Documentation/git-add.txt---chmod-x